src/style.css

```
@@ -0,0 +1,24 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.main_block {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+p {
+  margin: 0;
+}
```

```suggestion
h1,
p {
  margin: 0;
}
```

You are repeating yourself. You write `margin: 0` both in your `h1` and `p` rules. You may write as I've shown above and remove `margin: 0` from your `h1` selector